### ISSUE TITLE
fix search engine ordering

### ISSUE THREAD

**Issue by lime124** [mozilla-mobile/focus-android#1647]
#961 [meta] Ability to add search engines and or remove existing search engines

The current ordering seems arbitrary and will be difficult for people to find what they are looking for if the method of ordering isn't apparent. 
- Yahoo
- Google
- Amazon.com
- DuckDuckGo
- Twitter
- Wikipedia
See also #1618 for a screenshot.

It seems the current order is based on the order of entry in the xml file that contains the locale search engine list. 

I would advocate to use alphabetical ordering instead. And newly added search engines would fit in rather than be appended to the bottom of the list. This will make it much easier for people to quickly find the option they are looking for. 

Alphabetical ordering is also advocated for in the material guidelines: https://material.io/guidelines/components/lists.html#lists-behavior

@bbinto do you see any issue with this? Also, can we drop the .com in Amazon? It seems odd since the other don't use that.

**Comment by liuche** [comment]
re: Amazon.com, we can't change that because it's the name that BD gives us in the search list, and we can't change the name/logo/branding that is requested by the company. This is also the naming that's used across other Firefox products (Desktop, Firefox for Android). To change it we'd have to ask BD to ask Amazon, and that's probably too much work.

**Comment by bbinto** [comment]
Yeah I'd suggest not to tackle this as we've tried to do this in the past (changing orders) and Legal/BD wants us to keep it as is. Any change to the pre-defined list needs to go through their review.

**Comment by bbinto** [comment]
@lime124 let me know if you are ok with closing this

**Comment by lime124** [comment]
i'm following up on a couple things. i'll take responsibility for closing it :)

**Comment by lime124** [comment]
I should have updated this awhile ago but alas...I didn't. I did talk with Elvin from legal and he sad there was no legal requirement to keep them in a specific ordering. https://bugzilla.mozilla.org/show_bug.cgi?id=1416857

I then followed up with mconnor via slack and he said " tl;dr I’m not deeply worried about alphabetical order or not."

@brampitoyo to see it through to the finish line! idk why but this has been a pet peeve of mine that i will be happy to see resolved :D
